db/db_interface.go

```diff
@@ -4,4 +4,6 @@ type DB interface {
 	ListBuckets() []string
 	ListKeys(bucket string) []string
 	Get(bucket, key string) string
+	PrefixScan(bucket, prefix string, offset, limit int) []string
```

Comment:
PrefixScan and ListKeys are almost identical (prefix little bit better, more complete). Maybe just replace ListKeys with PrefixScan?

Reply:
ListKeys returns nothing or exactly one key. PrefixScan can return nothing, one or more keys.
They should not be replacements for each other.